src/components/StateBrowserNetwork.vue

```diff
@@ -287,6 +287,9 @@ export default {
             network.showServerBuffer('channels');
         },
         showBufferSettings(buffer) {
+            if (window.innerWidth < 770) {
```

Comment:
this needs to be `this.$state.ui.app_width <= 769`

Reply:
As of 2 minutes ago, this should now be `if (this.$state.ui.is_narrow) {`